app/js/handlebarsUserTable.js

```diff
@@ -19,10 +19,18 @@ function fillUserTable(HBTemplate) {
                     let html = template(userData)
                     user_list.innerHTML += html
                 })
+                let userItems = document.querySelectorAll(".user_item")
```

Comment:
are you sure it should be hooking on to the the whole div container? 

would it be better if you grabbed the button ID you made in the handle bar

Reply:
Solved thanks